release-team/role-handbooks/bug-triage/README.md

```diff
@@ -55,233 +33,208 @@ Bug Triage has a lower time requirement than most roles at the beginning of the
 
-### Issue Categorization
+As the PR is tagged for 1.17, is it still planned for this release?
+```
 
-Issues can be categorized with the following 'status codes', Green/Yellow/Red.
+#### Reports
 
-* green: issues/PRs where the resolution is known and will be resolved soon
-* yellow: issues/PRs which are in progress but are not as easy and need to be watched
-* red: issues/PRs that are heavily release-impacting, and/or with a likelihood of not making it in time.
+Beside the number of the `priority/critical-urgent` issues, you can start reporting the number of `priority/important-soon` issues and how good the response rate is for pinged issues.
 
-In each meeting, it is good to give an overall status of Bug Triage: Green, Yellow or Red, generally based on the amount of issues from each category. Examples are:
+Similar to the early release phase, the status (green/yellow/red) depends on the number of release blocking issues/PRs. You might want to declare yellow or red if there are big number of release blocking/`priority/critical-urgent` issues and/or if they're being blocked for any reason.
 
-- status can be Red if there are multiple release-impacting issues that are in danger of not being fixed in the release.
-- status can be Yellow if there are multiple issues open but they are being worked on, which is the usual state of the world.
-- status can be Green if there are no Red issues and workpaces seem normal.
+### Code Freeze
 
-It is useful to keep Red issues on a separate section and report them during the meetings.
+One of the main purposes of Code Freeze is to ensure that the code base is stable and that most of tests are passing (CI Signal is green). At this point, only release-blocking issues and PRs (usually bug fixes/`kind/bug`) are considered to be merged.
 
+Your responsibility here is to actively watch for any new issues/PRs targeting the milestone, monitor the status of existing issues/PRs, and filter the milestone by removing issues/PRs that are not needed for this release cycle.
 
-## Code Freeze
+#### Day Of The Code Freeze
 
-Once Code Freeze kicks in, only PRs targeting the upcoming release (/milestone v1.xx) are able to get merged. This is done to stabilize the release as much as possible, and reduce the overall tracking footprint to focus on the upcoming release.
+On the day of the Code Freeze, your responsibility is to try to help contributors to get the approval on their PRs and needed label. Check [How To Escalate](#how-to-escalate) part of the document for guide how to do this.
 
-This is enforced via test-infra automation, as a means of "new release coming in a few weeks - we're only merging relevant PRs in".
+When the code freeze starts, the highest priority has the PRs which had `approved` and `lgtm` labels before the code freeze started and are in the milestones. Depending on the merge queue length, it might be proposed to hold PRs that have `approved` and `lgtm`, but are not in the milestone.
```

Comment:
Query for this: [`is:pr is:open no:milestone label:approved label:lgtm`](https://github.com/kubernetes/kubernetes/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+no%3Amilestone+label%3Aapproved+label%3Algtm).